components/wso2is.key.manager.operations.endpoint/swagger.json

```diff
@@ -271,6 +271,95 @@
         "x-wso2-request" : "curl -X DELETE -H \"Authorization: Basic YWRtaW46YWRtaW4=\" -H \"Content-Type: application/json\" -d '{}' \"https://localhost:9443/keymanager-operations/dcr/register/s6BhdRkqt3\"\n",
         "x-wso2-response" : "\"HTTP/1.1 204 No Content\"\n"
       }
+    },
+    "/dcr/{clientId}/change-owner" : {
```

Comment:
why is this json file needed?

Reply:
Since this file is an auto-generated one, don't we need to commit them?

Reply:
Removed